src/Outsystems.SetupTools/Lib/PlatformSetup.ps1

```diff
@@ -450,26 +450,32 @@ function InstallDotNetCoreHostingBundle([string]$MajorVersion, [string]$Sources)
     }
 
     LogMessage -Function $($MyInvocation.Mycommand) -Phase 1 -Stream 2 -Message "Starting the installation"
-    $result = Start-Process -FilePath $installer -ArgumentList "/install", "/quiet", "/norestart" -Wait -PassThru -ErrorAction Stop
 
-    LogMessage -Function $($MyInvocation.Mycommand) -Phase 1 -Stream 2 -Message "Installation finished. Returnig $($result.ExitCode)"
-
-    return $($result.ExitCode)
+    if ($OnlyMostRecentHostingBundlePackage) {
```

Comment:
This flag is only for the uninstall old packages part, the installation of packages without the runtime should always be enabled by default.